frontend/components/App/App.tsx

```diff
@@ -69,9 +69,11 @@ const App = ({ children, location, router }: IAppProps): JSX.Element => {
       setAvailableTeams(available_teams);
       fetchConfig();
     } catch (error) {
-      console.log(error);
-      local.removeItem("auth_token");
-      window.location.href = "/login";
+      if (!location?.pathname.includes("/login/reset")) {
```

Comment:
```suggestion
      if (!location?.pathname.includes("/login")) {
```

So that we don't re-route anyone attempting any type of login logic.